versioned/storage/bigtable/src/main/java/org/projectnessie/versioned/storage/bigtable/BigTablePersist.java

```diff
@@ -565,6 +565,12 @@ private class ScanAllObjectsIterator extends AbstractIterator<Obj>
       this.filter = filter;
 
       Query q = Query.create(TABLE_OBJS).prefix(keyPrefix);
+
+      Filters.ChainFilter filterChain =
+          FILTERS
+              .chain()
+              .filter(FILTERS.key().regex(copyFromUtf8(config.repositoryId() + ":\\C*")));
```

Comment:
Same here, wouldn't it be slightly more efficient if we created a constant `ByteString SUFFIX = copyFromUtf8(":\\C*")`, then: 
```suggestion
              .filter(FILTERS.key().regex(copyFromUtf8(config.repositoryId()).concat(SUFFIX)));
```